Automattic/jetpack
# Jetpack: Render site logo in the QR Post when it exists

This PR renders the site logo in the center of the QR code when it exists.

Fixes https://github.com/Automattic/jetpack/issues/23289

#### Changes proposed in this Pull Request:

* Jetpack: Render site logo in the QR Post when it exists

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Set a site logo for your testing site
* Create a new post
* Publish
* Confirm you see the log in the center of the QR code
* Confirm it leads properly to the post page

https://user-images.githubusercontent.com/77539/157264326-77ae509c-7245-4715-9918-05b17cfc0a26.mov
